pkg/printers/internalversion/printers.go

```diff
@@ -642,7 +644,10 @@ func printPod(pod *api.Pod, options printers.PrintOptions) ([]metav1beta1.TableR
 		if nominatedNodeName == "" {
 			nominatedNodeName = "<none>"
 		}
-		row.Cells = append(row.Cells, podIP, nodeName, nominatedNodeName)
+		if len(pod.OwnerReferences) > 0 {
+			podOwner = fmt.Sprintf("%s/%s", pod.OwnerReferences[0].Kind, pod.OwnerReferences[0].Name)
```

Comment:
why is the first ownerReference chosen here? without the apiGroup, the kind alone is ambiguous.

Reply:
1.Generally, a pod only has one controller. multi controllers is not recommend. so i just chose the first ownerRef
2.This pr is for finding the controller of pod quickly,  kind + name ,i think this is enough. add more info , will reduce readability.

Reply:
this isn't checking if the selected ownerRef is even a controller… that seems the minimum required

Reply:
add it. PTAL, thanks!

Reply:
I find something wrong. i shou take the first ownerRef with 'controller=true'. i will fix it later , sorry!

Reply:
done! PTAL,thanks!